deploy/install/01-prerequisite/03-crd.yaml

```diff
@@ -2852,6 +2852,8 @@ spec:
                 type: string
               fromBackup:
                 type: string
+              restoreVolumeRecurringJob:
+                type: string
```

Comment:
Why not use boolean?

Reply:
This flag is used to override the globe setting, so it might not be able to be a boolean.   Now they are `"ignored"`, `"enabled"` and `"disabled"`.

Reply:
Got it. Thank you for the explaination.